GoogleDataTransport.podspec

```diff
@@ -29,6 +29,9 @@ Shared library for iOS SDK data transport needs.
   s.source_files = 'GoogleDataTransport/GDTCORLibrary/**/*'
   s.public_header_files = 'GoogleDataTransport/GDTCORLibrary/Public/*.h'
   s.private_header_files = 'GoogleDataTransport/GDTCORLibrary/Private/*.h'
+  s.ios.frameworks = 'SystemConfiguration', 'CoreTelephony'
+  s.osx.frameworks = 'SystemConfiguration', 'CoreTelephony'
+  s.tvos.frameworks = 'SystemConfiguration'
```

Comment:
Does watchOS need any?

Reply:
Both two framework is not available in watchOS, and if we update our api to adopt watchOS in the future, I think we will add relative framework for watchOS later.

Reply:
SG - thanks for checking.